component/commons/handler/src/main/java/org/wso2/carbon/identity/password/expiry/validation/handler/PasswordExpiryPolicyUtils.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2010-2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.password.expiry.validation.handler;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.model.IdentityProvider;
+import org.wso2.carbon.identity.application.common.model.IdentityProviderProperty;
+import org.wso2.carbon.identity.application.common.util.IdentityApplicationManagementUtil;
+import org.wso2.carbon.identity.event.IdentityEventConfigBuilder;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.bean.ModuleConfiguration;
+import org.wso2.carbon.idp.mgt.IdentityProviderManagementException;
+import org.wso2.carbon.idp.mgt.IdentityProviderManager;
+
+/**
+ * Read password expiry in days configuration.
+ */
+public class PasswordExpiryPolicyUtils {
+    private static final Log log = LogFactory.getLog(PasswordExpiryPolicyUtils.class);
+
+    private PasswordExpiryPolicyUtils(){}
+
+    /**
+     * Get identity event property for password expiry in days.
+     * @param tenantDomain
+     * @param propertyName
+     * @return
+     */
+    public static String getIdentityEventProperty(String tenantDomain, String propertyName) {
+
+        // Retrieving properties set in identity event properties
+        String propertyValue = null;
+        try {
+            ModuleConfiguration moduleConfiguration = IdentityEventConfigBuilder.getInstance()
+                    .getModuleConfigurations(PasswordExpiryValidationConstants.PASSWORD_CHANGE_EVENT_HANDLER_NAME);
+
+            if (moduleConfiguration != null) {
+                propertyValue = moduleConfiguration.getModuleProperties().getProperty(propertyName);
+            }
+        } catch (IdentityEventException e) {
+            //if identity property for password expiry in days can not be fetched, default value will be used.
+            log.warn("An error occurred while retrieving module properties");
+            if (log.isDebugEnabled()) {
+                log.debug("An error occurred while retrieving module properties because " + e.getMessage(), e);
```

Comment:
Since e is printed, e.getMessage is not necessary. Better to add some context information instead.